waku/v2/node/wakunode2.nim

```diff
@@ -685,6 +689,11 @@ when isMainModule:
 
     if conf.storenode != "":
       setStorePeer(node, conf.storenode)
+
+    #Log the Swap Account Balance just after the store node has been mounted and the peer has been credited
```

Comment:
The peer is only credited once the [`handler`](https://github.com/status-im/nim-waku/blob/master/waku/v2/protocol/waku_store/waku_store.nim#L342-L370) that is defined for `store` protocol is triggered, not simply after `store` is mount. The best place to log after `credit` is therefore in the `handler` itself.